ui/core/proto_utils/equipped_item.ts

```diff
@@ -25,6 +25,12 @@ export const getWeaponDPS = (item: Item, upgradeStep: ItemLevelState = ItemLevel
 	return (weaponDamageMin + weaponDamageMax) / 2 / (item.weaponSpeed || 1);
 };
 
+export const isThroneOfThunderWeapon = (item: Item) =>
+	[ItemType.ItemTypeWeapon, ItemType.ItemTypeRanged].includes(item.type) &&
+	item.phase == 3 &&
+	item.sources.some(itemSource => itemSource.source.oneofKind === 'drop' && itemSource.source.drop.zoneId === 6622);
```

Comment:
Do you need the additional check on phase when you're already checking the zone ID? Also, how will this interact with Celestial ToT LFR weapons?